blockstore/splitstore/splitstore.go

```diff
@@ -266,7 +268,13 @@ func Open(path string, ds dstore.Datastore, hot, cold bstore.Blockstore, cfg *Co
 	ss.txnViewsCond.L = &ss.txnViewsMx
 	ss.txnSyncCond.L = &ss.txnSyncMx
 	ss.chainSyncCond.L = &ss.chainSyncMx
-	ss.ctx, ss.cancel = context.WithCancel(context.Background())
+
+	baseCtx := context.Background()
+	ctx, err := tag.New(baseCtx, tag.Insert(metrics.Network, buildconstants.NetworkBundle))
+	if err != nil {
+		return nil, xerrors.Errorf("failed to create context with network tag: %w", err)
+	}
```

Comment:
you can use the `AddNetworkTag` to clean this up now, and also to skip the error (I'll comment inline)